src/keymap.cpp

```diff
@@ -401,14 +401,14 @@ static OpDesc opdescs[] = {
 	{
 		OP_SK_PGUP,
 		"pageup",
-		"PAGEUP",
+		"PPAGE",
```

Comment:
Just checking that I understand the purpose of this change:

1. the new key names come from STFL, right? (Well, they're also in our docs, but ultimately these are STFL's names)
2. the reason we need STFL's names is because we want to pass them into `bind` in STFL markup?
3. Newsboat worked fine despite the incorrect names because STFL handled page up/down by default, so the names we pass it were irrelevant?
4. now that this PR configures all the keys from within Newsboat, we absolutely need the defaults to use correct names?

Reply:
> The new key names come from STFL, right?

Yes, but STFL just passes through the value [given by ncurses' `keyname()`](https://github.com/dennisschagt/stfl/blob/cc355eb6a28e5ad25d20d5b83b57ad6ea1ada316/binding.c#L69). (STFL only [changes a few ascii values to a string](https://github.com/dennisschagt/stfl/blob/cc355eb6a28e5ad25d20d5b83b57ad6ea1ada316/binding.c#L32-L67))

> the reason we need STFL's names is because we want to pass them into bind in STFL markup?
> Newsboat worked fine despite the incorrect names because STFL handled page up/down by default, so the names we pass it were irrelevant?

Correct.
That part was broken because `PAGEUP` is not a valid key (it only worked because STFL used it's default keys when [`"**"` was specified](https://github.com/newsboat/newsboat/blob/81cb6b6b33c5e1d8be015ed235979183f6b85009/src/view.cpp#L105-L133))

> now that this PR configures all the keys from within Newsboat, we absolutely need the defaults to use correct names?

Yes, otherwise we would ignore `"PPAGE"` and `"NPAGE"` as they would not be present in the keymap.